### ISSUE TITLE
Uneeded parameters being passed when exploring

### ISSUE THREAD

**Issue by prury** [kodadot/nft-gallery#9549]
i think some unneeded parameters are being passed if you filter and click on an nft on explorer

`?listed=true`

https://github.com/kodadot/nft-gallery/assets/36627808/203af806-cdd3-4903-9893-df07afb48da8

**Comment by exezbcz** [comment]
yes, its because there was a small hack to show listed when selecting sorting filter: low to high
